pages/Transfer.qml

```diff
@@ -528,82 +528,199 @@ Rectangle {
-                    root.sweepUnmixableClicked()
+                MoneroComponents.Label {
+                    id: offlineTxSignLabel
+                    color: MoneroComponents.Style.defaultFontColor
+                    text: qsTr("Offline transaction signing") + translationManager.emptyString
+                    fontSize: 14
+                    wrapMode: Text.WordWrap
                 }
             }
-
-            StandardButton {
-                id: saveTxButton
-                text: qsTr("Create tx file") + translationManager.emptyString
-                visible: appWindow.viewOnly
-                enabled: pageRoot.checkInformation(amountLine.text, addressLine.text, appWindow.persistentSettings.nettype)
-                small: true
-                onClicked: {
-                    console.log("Transfer: saveTx Clicked")
-                    var priority = priorityModelV5.get(priorityDropdown.currentIndex).priority
-                    console.log("priority: " + priority)
-                    console.log("amount: " + amountLine.text)
-                    addressLine.text = addressLine.text.trim()
-                    setPaymentId(paymentIdLine.text.trim());
-                    root.paymentClicked(addressLine.text, paymentIdLine.text, amountLine.text, root.mixin, priority, descriptionLine.text)
-
+            
+            ColumnLayout {
+                Layout.fillWidth: true
+                spacing: 12
+            
+                RowLayout {
+                    Layout.fillWidth: true
+                    spacing: 12
+                    
+                    StandardButton {
+                        id: saveTxButton
+                        text: qsTr("1. Create") + translationManager.emptyString
+                        visible: true 
+                        enabled: appWindow.viewOnly && pageRoot.checkInformation(amountLine.text, addressLine.text, appWindow.persistentSettings.nettype)
+                        small: true
+                        onClicked: {
+                            console.log("Transfer: saveTx Clicked")
+                            var priority = priorityModelV5.get(priorityDropdown.currentIndex).priority
+                            console.log("priority: " + priority)
+                            console.log("amount: " + amountLine.text)
+                            addressLine.text = addressLine.text.trim()
+                            setPaymentId(paymentIdLine.text.trim());
+                            root.paymentClicked(addressLine.text, paymentIdLine.text, amountLine.text, root.mixin, priority, descriptionLine.text)
+                        }
+                    }
+    
+                    StandardButton {
+                        id: signTxButton
+                        text: qsTr("2. Sign (offline)") + translationManager.emptyString
+                        small: true
+                        visible: true
+                        enabled: !appWindow.viewOnly && appWindow.disconnected
```

Comment:
I think this means disconnected from the daemon, not no internet connection. Is this how you intended it?

Reply:
We can infer that the computer is online if the GUI is connected to the daemon. Not sure if there is a better way to detect if the computer is online. 

Also, to be a true offline (cold) storage, it would be nice to have a variable that stores if the spend wallet has ever been connected to the daemon/internet. If this variable is true, then sign button should be disabled.